plugins/ipam/host-local/backend/disk/backend.go

```diff
@@ -124,3 +173,10 @@ func GetEscapedPath(dataDir string, fname string) string {
 	}
 	return filepath.Join(dataDir, fname)
 }
+
+func UnescapePath(fname string) net.IP {
```

Comment:
Asymmetry between `GetEscapedPath` and `UnescapePath `.
Maybe `EscapedPathToIP` ?